docker/1.4.8/README.md

```diff
@@ -42,7 +42,7 @@ docker build -t mosquitto:1.4.8 .
 ##Run
 Run a container using the new image:
 ```
-docker run -it -p 1883:1883 -p 9001:9001 -v mosquitto.conf:/mosquitto/data/mosquitto.conf -v /mosquitto/data -v /mosquitto/log mosquitto:1.4.8
+docker run -it -p 1883:1883 -p 9001:9001 -v `pwd`/mosquitto.conf:/mosquitto/config/mosquitto.conf -v /mosquitto/data -v /mosquitto/log mosquitto:1.4.8
```

Comment:
This would mean that nothing gets mounted in the container though, right? Three data volumes would get created inside the container only: `/mosquitto/config/mosquitto.conf`, `/mosquitto/data` and `/mosquitto/log`.

This is fine, but would mean that the user's custom config would not be used at container start. We need to explicitly mount the host file in the container data volume using:

```